framework/JobHandler.py

```diff
@@ -127,6 +127,11 @@ def initialize(self,runInfoDict,messageHandler):
     """
     self.runInfoDict = runInfoDict
     self.messageHandler = messageHandler
+    self.maxQueueSize = runInfoDict.get('maxQueueSize',runInfoDict.get('batchSize'))
```

Comment:
the ```maxQueueSize``` is predefined in line 83, please remove that.

Reply:
I partially agree.  Providing a placeholder in the `__init__` function is a developer-conscious way to define the variable in a place that's straightforward to find.

However, it should not be initialized to a value, so I set it to `None`.